user-data-store-server/src/main/java/com/wultra/security/userdatastore/service/PhotoService.java

```diff
@@ -104,6 +106,23 @@ public PhotoCreateResponse createPhoto(final PhotoCreateRequest request) {
         return new PhotoCreateResponse(photoEntity.getId(), documentEntity.getId());
     }
 
+    @Transactional
+    public Response updatePhoto(final String photoId, final PhotoUpdateRequest request) {
```

Comment:
Could you also implement update of the `timestampLastUpdated` of the corresponding document, when a photo is changed?
It would be useful to be able to find out, if there was any change made without the need of fetching a large image.

Reply:
Ah, yes, I will add it.